### ISSUE TITLE
iOS Feature Request: Keyboard shortcut and gesture to close image

### ISSUE THREAD

**Issue by benfsmith** [brentsimmons/NetNewsWire#2954]
Currently, when viewing an image in full screen view, I think the only ways to dismiss or close are to click on the X circle icon or swipe on screen. It would be helpful to also have a keyboard shortcut (e.g., command + w), and a trackpad and/or magic mouse gesture (e.g., one-finger swipe up or down). Sorry I don’t know how develop the feature myself. Thank you for considering

**Comment by benfsmith** [comment]
That would be a screen gesture, not a trackpad or mouse gesture, right?

**Comment by kevinm6** [comment]
> That would be a screen gesture, not a trackpad or mouse gesture, right?

No, is a macOS gesture, even in Photos app on Mac you can dismiss the image view with swipe down on trackpad or mouse.
But I think could be nice add this feature with the "Esc" key on macOS keyboards

**Comment by benfsmith** [comment]
Right. Per title, was referring to iOS, though iPadOS would have been more specific. On Mac NNW, I don't think images open full screen, right?

**Comment by kevinm6** [comment]
Yes, my fault, sorry!
Not for now, if I remember well is impossible even to use QuickLook on media in NNW

**Comment by brentsimmons** [comment]
I’m a little confused — what’s the specific bug or feature request here?

**Comment by benfsmith** [comment]
It's a feature request for a way to dismiss an opened image in NNW for iPadOS using a magic keyboard or other external keyboard or trackpad. For example command + . (esc) and/or to be able to swipe the image away to reflect the main screen or article again. Does that make sense? Somewhat related, it would be great if NNW for Mac could open article images as is possible on iPadOS
